source/administration/devops-command-center.rst

```diff
@@ -3,199 +3,255 @@ Mattermost Incident Management
+- ``/incident commander [@username]`` - Show or change the current commander.
+- ``/incident info`` - Show a summary of the current incident.
+- ``/incident stage [next/prev]`` - Move to the next or previous stage.
 
-You can also:
+Generating test data
+^^^^^^^^^^^^^^^^^^^^
 
-- Assign a step to yourself or another incident member.
-- Mark a step as **Complete** or **Incomplete**.
-- Start an automated action.
-- Invite new members to the channel.
+To quickly Mattermost Incident Management, use the following test commands to create incidents populated with random data:
+- ``/incident test create-incident [playbook ID] [timestamp] [incident name]`` - Provide the ID of an existing playbook to which the current user has access, a timestamp, and an incident name. The command creates an ongoing incident with the creation date set to the specified timestamp.
+  * An example command looks like: ``/incident test create-incident 6utgh6qg7p8ndeef9edc583cpc 2020-11-23 PR-Testing``
+- ``/incident test bulk-data [ongoing] [ended] [start date] [end date] [seed]`` - Provide a number of ongoing and ended incidents, a start and end date, and an optional random seed. The command creates the given number of ongoing and ended incidents, with creation dates randomly between the start and end dates. The seed may be used to reproduce the same outcome on multiple invocations. Incident names are generated randomly. 
+  * An example command looks like: ``/incident test bulk-data 10 3 2020-01-31 2020-11-22 2``
 
 Channel Export
 --------------
 
-Please see the `Channel Export plugin documentation <https://mattermost.gitbook.io/channel-export-plugin>`_ for more information.
-
-Glossary
---------
-
-- **Commander:** The Mattermost user currently responsible for transitioning an incident from ongoing to ended.
-- **Incident:** An event requiring the coordinated actions of one or more Mattermost users. An incident is either ongoing or ended.
-- **Incident channel:** A Mattermost channel dedicated to real-time conversation about the incident.
-- **Incident insight page:** The incident details and analytics page, which also provides the channel export download link. It is not available on mobile.
-- **Incident member:** A Mattermost user with access to the corresponding incident channel.
-- **Playbook:** A task-based process that's followed in order to resolve an incident.
-- **Playbook configuration page:** The playbook configuration and editing page. It is not available on mobile.
-- **Stage:** A set of tasks grouped together to achieve a specific goal of the workflow, which generally need to be completed before proceeding to the next stage of the incident resolution process.
-- **Tasks:** The individual steps required to complete the stages of an incident. Tasks can optionally be assigned to specific incident participants into stages.
-- **The RHS:** The incident list and incident details displayed on the right hand side (RHS) of the web app. It is not available on mobile.
```

Comment:
If this page will contain a Glossary that explains the RHS, then please feel free to ignore the suggestions where I've written out the acronym in full.

Reply:
The glossary has been removed so I've committed your suggestions. :)